Content.Client/Administration/UI/Tabs/ObjectsTab/ObjectsTab.xaml.cs

```diff
@@ -81,32 +111,63 @@ private void RefreshObjectList(ObjectsTabSelection selection)
                 throw new ArgumentOutOfRangeException(nameof(selection), selection, null);
         }
 
-        foreach (var control in _objects)
-        {
-            ObjectList.RemoveChild(control);
-        }
+        var sortedEntities = _ascending
+            ? entities.OrderBy(e => GetComparableValue(e, _headerClicked)).ToList()
+            : entities.OrderByDescending(e => GetComparableValue(e, _headerClicked)).ToList();
```

Comment:
nevermind, populate sortedEntities in a loop with these checks and the check in SearchList.PopulateList